sdk/monitor/azure-monitor-query/swagger/log_query_swagger.json

```diff
@@ -1,7 +1,7 @@
 {
   "swagger": "2.0",
   "info": {
-    "version": "v1",
+    "version": "2021-05-19_Preview",
```

Comment:
Did the service change their API versioning scheme? ACR is in the similar boat, did you work with them to use the newer scheme here?

Reply:
I authored the swagger for this preview, so, I used the correct format here. I am waiting for feedback from the service team to see if they are okay with the swagger version change.